ci/scp-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -eu
+
+# Expected environment variables:
+# TRAVIS_REPO_SLUG, TRAVIS_TAG
+# STORAGE_USER, STORAGE_PRIVATE_KEY, STORAGE_HOST, STORAGE_PATH
+
+RELEASE_FILE_CONTENT_TYPE="audio/mpeg"
+STORAGE_PRIVATE_KEY_FILE="storage.key"
+
+# Use GitHub API to get the release file name and URL associated with a Git tag.
+
+RELEASE_JSON="$(curl \
+  --silent \
+  --fail \
+  --show-error \
+  --request GET \
+  --header "Accept: application/vnd.github.v3+json" \
```

Comment:
Let's also add retries (curl supports that), when I was testing one time it took really long and I was afraid it would timeout